### PR TITLE
Add Go to the image

### PR DESCRIPTION
**Pull request by davemackintosh**
Install Go so that CDK applications written in Go can be built.

### CODE REVIEW

**Comment by davemackintosh** [comment]
I'll fix the linting issues and I'll squash these commits.

**Comment by ScottBrenner** [comment]
Do you have an example workflow run of the Action executing with these changes?

**Comment by davemackintosh** [comment]
Sorry, just remembered about this. I'll put together an example repo later on today for you.

**Comment by ScottBrenner** [comment]
Thanks - please edit https://github.com/ScottBrenner/aws-cdk-action/blob/master/README.md#github-action-for-aws-cdk to include a mention of Go support as well

File: Dockerfile [hunk 1]
```diff
@@ -12,10 +12,17 @@ LABEL "com.github.actions.color"="yellow"
 
 LABEL "maintainer"="Scott Brenner <scott@scottbrenner.me>"
 
-RUN apk --no-cache add nodejs npm python3 py3-pip
+RUN apk --no-cache add nodejs npm python3 py3-pip git make musl-dev go bash
```
Comment (ScottBrenner) [inline Dockerfile:15]:
```suggestion
RUN apk --no-cache add nodejs npm python3 py3-pip git make musl-dev go
```
Is bash necessary here?

File: entrypoint.sh [hunk 2]
```diff
@@ -6,4 +6,5 @@ if [ -f "requirements.txt" ]; then
   pip3 install -r requirements.txt
 fi
 
+git config --global --add safe.directory /github/workspace
```
Comment (ScottBrenner) [inline entrypoint.sh:9]: What is this being set for?

Comment (davemackintosh) [reply entrypoint.sh:9]: Sorry for the incredibly slow reply, busy at work and life. Modern versions of Git require explicit knowledge of "safe" directories due to a CVE. Given this is our own workspace, this should be fine. https://github.blog/2022-04-12-git-security-vulnerability-announced/#cve-2022-24765

**Review by ScottBrenner — APPROVED** [review]

**Comment by ScottBrenner** [comment]
Thank you for contribution! Released in https://github.com/ScottBrenner/aws-cdk-action/releases/tag/v1.6.0

**Pull request merged**